root/types.js

```diff
@@ -16,7 +16,7 @@
  */
 
 declare type AggregatedTagT = {|
-  +tag: string,
+  +tag: TagT,
   +count: number,
```

Comment:
Should _last updated_ be moved here instead of deleted?

Reply:
Why? It AggregatedTag doesn't seem to have a last_updated, or am I missing something?

Reply:
I probably confused it with something else, cannot recall what.